ddev/changelog.d/21587.added

```diff
@@ -0,0 +1 @@
+Added debugging logs to `ddev size`
```

Comment:
I would say this PR introduces a lot of changes than just adding debugging logs. Functions signature, and a lot of code logic has been altered.